app/grandchallenge/uploads/widgets.py

```diff
@@ -27,6 +27,14 @@ class Media:
         )
 
 
+class DICOMUserUploadMixin(UserUploadWidgetMixin):
+    class Media(UserUploadWidgetMixin.Media):
+        js = UserUploadWidgetMixin.Media.js + (
```

Comment:
This doesn't look correct to me, would it work with multiple mixins (e.g. like in `DICOMUserUploadMultipleWidget` if `MultipleHiddenInput` also defined `Media`)? Is this the recommended way to do it in Django? 

Why is this a mixin anyway? It is only used once, so maybe move it to the concrete implementation and if this is the recommended way ensure that `Media` inherits both subclasses?